### PR TITLE
Hotfix 0.16.1.1

### PR DESCRIPTION
**Pull request by UdjinM6**

### CODE REVIEW

**Comment by UdjinM6** [comment]
In case you are wondering why don't we backport #3816 in a separate PR, I'm planning to merge this PR without squashing and expect this link https://github.com/dashpay/dash/pull/3818/commits/1734872895fb989904b219b133d66aa460d157a9#diff-e04cb1e9f29f76897c6b84334238f621ff45ba13cf180ad1d0d442d50ebe51f7R61 (https://github.com/dashpay/dash/commit/ccef3b48363d8bff4b919d9119355182e3902ef3) to be a valid one because of that. I hope I'm not missing anything :)

File: doc/release-notes.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+See detailed [set of changes](https://github.com/dashpay/dash/compare/v0.16.1.0...dashpay:v0.16.1.1).
+
+- [`ccef3b4836`](https://github.com/dashpay/dash/commit/ccef3b48363d8bff4b919d9119355182e3902ef3) qt: Fix wallet encryption dialog (#3816)
+
+Credits
+=======
+
+Thanks to everyone who directly contributed to this release:
+
+- UdjinM6
```
Comment (PastaPastaPasta) [inline doc/release-notes.md:68]: You thanking yourself :D

**Review by PastaPastaPasta — APPROVED** [review]
utACK, release notes look good. Everything looks as expected

**Review by xdustinface — APPROVED** [review]
utACK

**Comment by PastaPastaPasta** [comment]
https://github.com/dashpay/dash/commit/ccef3b48363d8bff4b919d9119355182e3902ef3 looks good

**Pull request merged**